### PR TITLE
Use sed to trim whitespace

### PR DESCRIPTION
**Pull request by baijum**
Fix issue #66

Based on the SO answer: https://stackoverflow.com/a/3232433/547840

### CODE REVIEW

File: scripts/src/chartprreview/verify-report.sh [hunk 1]
```diff
@@ -141,7 +141,7 @@ getAnnotations() {
 
 
   while IFS= read -r line; do
-    line=`echo $line | xargs`
+    line="$(echo -e "${line}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
```
Comment (pedjak) [inline scripts/src/chartprreview/verify-report.sh:144]: we are reading here yaml structure, and reading it is line by line is very fragile and can catch things on e.g. wrong level. We should either use tools like [yq](https://github.com/mikefarah/yq) or perhaps this script should be written in Python or some other language with better yaml handling?

**Review by pedjak — CHANGES_REQUESTED** [review]

File: scripts/src/chartprreview/verify-report.sh (hunk 1 above)
Comment (baijum) [reply scripts/src/chartprreview/verify-report.sh:144]: I am fine with converting this script to Python. For now, let's merge this PR.

@mmulholla Any thoughts?

Comment (pedjak) [reply scripts/src/chartprreview/verify-report.sh:144]: ok, if this unblock you for a moment, but let's create an issue addressing this.

**Review by pedjak — APPROVED** [review]

File: scripts/src/chartprreview/verify-report.sh (hunk 1 above)
Comment (baijum) [reply scripts/src/chartprreview/verify-report.sh:144]: I have created an issue here: #74

**Pull request merged**